### PR TITLE
Adding robot xacros information

### PR DESCRIPTION
**Pull request by mikramarc**
## Proposed changes

Adding robot xacros information.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [x] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [x] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: docs/user_guide/3_robot_xacros.rst [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+Unimanual
+~~~~~~~~~~
+
+The main xacro file to use is `sr_hand.urdf.xacro <https://github.com/shadow-robot/sr_common/blob/F_new_xacros_ready/sr_description/robots/sr_hand.urdf.xacro>`_ when you are using only one of our hands.
+
+The following arguments are available:
+
+* ``side`` - defines the side of the hand. Allowed options: ``right``/``left``
+* ``hand_type`` - defines the type of the hand. Allowed optins: ``hand_e``/``hand_g``/``hand_c``
```
Comment (bdenoun) [inline docs/user_guide/3_robot_xacros.rst]: optins --> options

File: docs/user_guide/3_robot_xacros.rst [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
++=============+=====+=========+========+
+| tip_sensors | pst |  bt_2p  |  bt_sp |
++-------------+-----+---------+--------+
+
+Bimanual
+~~~~~~~~~
+
+If you have a setup with two robot hands, this is the xacro to use: `sr_hand_bimanual.urdf.xacro <https://github.com/shadow-robot/sr_common/blob/F_new_xacros_ready/sr_description/robots/sr_hand_bimanual.urdf.xacro>`_
+
+The following arguments are available (similar to the hand only scenario but with the side prefix to specify every configuration):
```
Comment (bdenoun) [inline docs/user_guide/3_robot_xacros.rst]: hand only --> hand-only?

**Review by BenStarmerSmith — APPROVED** [review]

**Review by bdenoun — CHANGES_REQUESTED** [review]

File: docs/user_guide/3_robot_xacros.rst (hunk 1 above)
Comment (mikramarc) [reply docs/user_guide/3_robot_xacros.rst]: fixed in 0420951

File: docs/user_guide/3_robot_xacros.rst (hunk 2 above)
Comment (mikramarc) [reply docs/user_guide/3_robot_xacros.rst]: fixed in 0420951

**Review by bdenoun — APPROVED** [review]

File: docs/user_guide/3_robot_xacros.rst [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+     roslaunch sr_robot_launch srhand.launch side:=right hand_type:=hand_g hand_version:=G1M5 fingers:=th,ff,mf,rf,lf tip_sensors:=ff=bt_2p,lf=bt_sp,mf=pst,rf=pst,th=bt_sp mid_sensors:=none prox_sensors:=none palm_sensor:=none sim:=true
+
+* Xacro command:
+
+  .. prompt:: bash $
+
+     xacro <xacro file> side:=right hand_type:=hand_g hand_version:=G1M5 fingers:=th,ff,mf,rf,lf tip_sensors:=ff=bt_2p,lf=bt_sp,mf=pst,rf=pst,th=bt_sp mid_sensors:=none prox_sensors:=none palm_sensor:=none
+
+As far as SRDF’s are concerned, all necessary ones are autogenerate from ``robot_description`` ros parameters spawned to the parameter server.
```
Comment (maxzieba) [inline docs/user_guide/3_robot_xacros.rst]: autogenerated

**Pull request merged**